### PR TITLE
Run `pnpm dedupe`

### PR DESCRIPTION
**Pull request by auscompgeek**
#1745 is almost impossible to review :sweat_smile:

I'd suggest running `pnpm dedupe` on the PR but it turns out `main` has a whole bunch of duplicate transitive deps too, so running it against `main` first.

## Checklist

- [ ] ~I have added [tests](https://www.cursorless.org/docs/contributing/test-case-recorder/)~
- [ ] ~I have updated the [docs](https://github.com/cursorless-dev/cursorless/tree/main/docs) and [cheatsheet](https://github.com/cursorless-dev/cursorless/tree/main/cursorless-talon/src/cheatsheet)~
- [ ] I have not broken the cheatsheet

### CODE REVIEW

**Review by pokey — APPROVED** [review]

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -8117,6 +7991,7 @@ packages:
 
   /debuglog@1.0.1:
     resolution: {integrity: sha512-syBZ+rnAK3EgMsH2aYEOLUW7mZSY9Gb+0wUMCFsZvcmiz+HigA0LOcq/HoQqVuGG+EKykunc7QG2bzrponfaSw==}
+    deprecated: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
```
Comment (auscompgeek) [inline pnpm-lock.yaml:7994]: Looks like this is a dependency of [`readdir-scoped-modules@1.1.0`](https://www.npmjs.com/package/readdir-scoped-modules) which in itself is also deprecated.

> This functionality has been moved to @npmcli/fs

File: pnpm-lock.yaml [hunk 2]
```diff
@@ -15932,6 +15786,7 @@ packages:
   /typedoc-plugin-resolve-crossmodule-references@0.3.3(typedoc@0.23.28):
     resolution: {integrity: sha512-ZWWBy2WR8z9a6iXYGlyB3KrpV+JDdZv1mndYU6Eh6mInrfMCrQJi3Y5K9ihMBfuaBGB//le1nEmQLgzU3IO+dw==}
     engines: {node: '>=14'}
+    deprecated: Upgrade to typedoc >= 0.24 and remove typedoc-plugin-resolve-crossmodule-references from your dependencies
```
Comment (auscompgeek) [inline pnpm-lock.yaml:15789]: New deprecation for a direct dependency :eyes:

Comment (pokey) [reply pnpm-lock.yaml:15789]: yeah typedoc has been a bit of a nightmare tbh.  i'm tempted to switch off our api docs, as I don't think anyone is using them anyways 🤔

**Comment by auscompgeek** [comment]
> I thought pnpm deduped by default nowadays?

:shrug: I don't think it'll try to update existing locked transitive dependencies, but will reuse existing resolutions. Both pnpm and [yarn behave this way](https://yarnpkg.com/cli/dedupe/) I think.

File: pnpm-lock.yaml (hunk 1 above)
Comment (pokey) [reply pnpm-lock.yaml:7994]: appears to be mainly coming from webpack, so maybe we bump webpack and see if that fixes?

Comment (auscompgeek) [reply pnpm-lock.yaml:7994]: So it seems! Looks like webpack is a dependency of docusaurus too, not just a direct `devDependency` for the cheatsheet.

Comment (pokey) [reply pnpm-lock.yaml:7994]: https://github.com/cursorless-dev/cursorless/issues/1748

File: pnpm-lock.yaml (hunk 2 above)
Comment (pokey) [reply pnpm-lock.yaml:15789]: https://github.com/cursorless-dev/cursorless/issues/1749

**Pull request merged**